_includes/_acquire.htm

```diff
@@ -13,28 +13,34 @@ <h4 class="text-bold">
             <div class="col-xs-12">
                 <ul>
                     <li>
-                        Prior to being able to stake or transfer Gridcoin to/from the Gridcoin client you need to fully sync the Gridcoin client to the latest block on the blockchain (green tick under connections indicates a fully synced wallet)
+						Before your wallet can do anything, it needs to be synced (the green tick under connections means you are synced)
                     </li>
                     <li>
                         If you don't want to sync from 0, then you can download a snapshot of the blockchain <a href="https://download.gridcoin.us/download/downloadstake/signed/snapshot.zip">here</a>.
                         <ul>
-                             <li>
-                                Note: The snapshot will not fully sync your client, but will rather bring you up to 80-90% sync. Allow your client to manually sync after the snapshot has been implemented.
+                            <li>
+								To use, open the folder (Linux = .GridcoinResearch/, Windows = %appdata%\GridcoinResearch) and replace all files from the snapshot <b>except wallet.dat and gridcoinresearch.conf</b>
+							</li>
+							<li>
+                                The snapshot will only bring you up to 80-90% sync. Allow it to continue syncing after the snapshot has been implemented.
                             </li>
                         </ul>
                     </li>
                     <li>
                         If you are unable to acquire connections to the Gridcoin network, attempt the following:
                         <ul>
                             <li>
-                                Verify that your computer time is synced to your configured time server.
+                                Verify you're running the latest version of the Gridcoin client, as there are frequent updates. Updates are posted on the <a href="https://github.com/gridcoin-community/Gridcoin-Research/releases">GitHub Release Page</a>.
                             </li>
-                            <li>
-                                Investigate whether or not your ISP is blocking P2P traffic.
+							<li>
+                                Verify that your computer's time is in the correct timezone.
```

Comment:
This could just say 'Verify that your computer's time is correct' as it's not a matter of timezone being right but the time being synced with the ntp server. Could also be worded 'Verify that your computer's clock is correct' Probably more understandable to the average person.

Reply:
That's where the issue is 99% of the time since you have to drift 90 minutes out from the network time so it's hard to have that be the case unless the timezone is wrong. From having helped people with this issue, they don't do that unless they're told to do that.

Reply:
If that's the case then it's fine as is. Good thing is either way it'll make users check their time regardless.